diff --git a/lib/fko_encode.c b/lib/fko_encode.c
index c92d421..61e8645 100644
--- a/lib/fko_encode.c
+++ b/lib/fko_encode.c
@@ -99,6 +99,8 @@ fko_encode_spa_data(fko_ctx_t ctx)
     if(tbuf == NULL)
         return(FKO_ERROR_MEMORY_ALLOCATION);
 
+    ctx->rand_val[2] = 0x10;
+
     /* Put it together a piece at a time, starting with the rand val.
     */
     strlcpy(tbuf, ctx->rand_val, FKO_ENCODE_TMP_BUF_SIZE);
